pydrumscore/export_files/META-INF/container.xml

```diff
@@ -0,0 +1,9 @@
+<container>
+  <rootfiles>
+    <rootfile full-path="King Nothing.mscx" />
+    <rootfile full-path="Thumbnails/thumbnail.png" />
+    <rootfile full-path="viewsettings.json" />
+    <rootfile full-path="score_style.mss" />
+    <rootfile full-path="audiosettings.json" />
+    </rootfiles>
+  </container>
```

Comment:
Writing this here because GitHub doesn't let me comment on an image: the thumbnail.png is an image for a specific song ("Highway to Hell"). We should just either put a dummy or not add a thumbnail at all. I would hope that MuseScore just handles that case and displays nothing.

When we have a finished logo, we can maybe add it in the thumbnails. 😌